conbench/__init__.py

```diff
@@ -3,6 +3,8 @@
 import logging
 import os
 
+from prometheus_flask_exporter.multiprocess import GunicornInternalPrometheusMetrics
```

Comment:
As this stands, I think you'll need to add `prometheus_flask_exporter` to `requirements-cli.txt` as well, because this is the CLI's `__init__.py` too. Then we'll need to fix the [conda feedstock](https://github.com/conda-forge/conbench-feedstock). 

Or maybe you can trycatch this and do nothing if `prometheus_flask_exporter` isn't installed?

Reply:
Ha! Thanks for catching that! Will think about this tomorrow. 

Looking forward to having more separation of concerns.

Reply:
I added a commit to deal with this, and also left a note in `conbench/__init__.py` to warn about the fact that this is used in two rather different contexts.